WordPress/twentytwentyfour
# Add code style config files

Copy & paste from twentytwentythree to get started.

Ref. https://github.com/WordPress/twentytwentyfour/issues/5


### Testing instructions

1. Install node and composer, and then project dependencies:
```sh
npm install
composer install
```
2. Check that linting commands works. The ones available are:

- `npm run lint:css` lints and autofixes where possible the CSS
- `composer run analyze [filename.php]` statically analyzes PHP for bugs
- `composer run lint` checks PHP for syntax errors
- `composer run standards:check` checks PHP for standards errors according to [WordPress coding standards](https://developer.wordpress.org/coding-standards/)
- `composer run standards:fix` attemps to automatically fix errors

In this branch, there are no errors; to test, you could add errors, for example, in pattern files by messing with the indentation or spaces.
